protocol-configuration/dependencies.ts

```diff
@@ -924,7 +926,24 @@ const dependencies: DependencyMap = {
   },
   pegExchanger: {
     contractDependencies: ['tribe', 'feiDAOTimelock']
+  },
+  weightedBalancerPoolManagerBase: {
+    contractDependencies: []
   }
+  /*balancerDepositFeiWeth: {
+    contractDependencies: [
+      'core',
+      'balancerVault',
+      'balancerRewards',
+      'balancerFeiWethPool',
+      'weth',
+      'oneConstantOracle',
+      'chainlinkEthUsdOracleWrapper'
+    ]
+  },
+  delayedPCVMoverWethUniToBal: {
+    contractDependencies: ['core', 'uniswapPCVDeposit', 'balancerDepositFeiWeth', 'ratioPCVControllerV2']
+  }*/
```

Comment:
remove commented deps

Reply:
will uncomment after deployment because these need to be in the file after the contracts are in mainnetAddresses.ts